packages/service-library/tests/long_running_tasks/test_long_running_tasks_task.py

```diff
@@ -29,12 +30,12 @@
 from tenacity.stop import stop_after_delay
 from tenacity.wait import wait_fixed
 
-_RETRY_PARAMS = dict(
-    reraise=True,
-    wait=wait_fixed(0.1),
-    stop=stop_after_delay(60),
-    retry=retry_if_exception_type(AssertionError),
-)
+_RETRY_PARAMS: dict[str, Any] = {
+    "reraise": True,
+    "wait": wait_fixed(0.1),
+    "stop": stop_after_delay(60),
+    "retry": retry_if_exception_type(AssertionError),
```

Comment:
If you use `TryAgain` it is cleaner since `assert expression` also raises `AssertionError`